opentreemap/treemap/js/src/buttonEnabler.js

```diff
@@ -52,9 +55,15 @@ function fullyEnable($el, href) {
 }
 
 function fullyDisable($el, disabledTitle) {
+    var wrapperTemplate =
+            '<label ' + disabledButtonWrapperAttrPair +
+            ' data-placement="bottom" title="%s"></label>';
```

Comment:
Ah yeah, when I pulled out the bootstrap portion I left that there. Will remove.